sdk/webpubsub/azure-messaging-webpubsubclient/azure/messaging/webpubsubclient/models/_models.py

```diff
@@ -402,29 +615,57 @@ def write_message(message: WebPubSubMessage) -> str:
 
 
 class WebPubSubJsonProtocol(WebPubSubClientProtocol):
+    """WebPubSub json protocol"""
+
     def __init__(self) -> None:
         super().__init__()
         self.is_reliable_sub_protocol = False
         self.name = "json.webpubsub.azure.v1"
 
 
 class WebPubSubJsonReliableProtocol(WebPubSubClientProtocol):
+    """WebPubSub json reliable protocol"""
+
     def __init__(self) -> None:
         super().__init__()
         self.is_reliable_sub_protocol = True
         self.name = "json.reliable.webpubsub.azure.v1"
 
 
 class SendMessageErrorOptions:
```

Comment:
I'm not sure whether the `Options` suffix makes sense as this class is more like an entity of `ackMap`?

Reply:
It is not exposed to users and I prefer current name